packages/arb-avm-cpp/cavm/caggregator.h

```diff
@@ -52,10 +52,18 @@ typedef struct CRequestInfoStruct CRequestInfo;
 void deleteAggregatorStore(CAggregatorStore* m);
 
 Uint64Result aggregatorBlockCount(const CAggregatorStore* agg);
-int aggregatorSaveBlock(CAggregatorStore* agg,
```

Comment:
Need to remove `aggregatorSaveBlockHash` and `aggregatorSaveRequest` as well

Reply:
done